about.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Science Festival Site</title>
+    <link rel="stylesheet" href="assets/css/style.css" />
+  </head>
+  <body>
+    <!-- HEADER -->
+    <header class="header">
+      <div>
+        <a href="index.html"><img src="assets/images/logo_blanc.png" class="logo" alt="Link to return to the homepage" /></a>
+      </div>
+      <div class="open_menu">
+        <img src="assets/images/menu.png" alt="Open the menu" />
+      </div>
+      <nav class="menu hidden">
+        <div class="close_menu"><img src="assets/images/cancel.png" alt="Close the menu" /></div>
+        <ul class="header_nav_list">
+          <li><a class="homepage hidden" href="index.html">Homepage</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li class="campagne"><a href="#">Campagne</a></li>
+        </ul>
+      </nav>
+    </header>
+    <!-- HEADER END -->
+    <!-- HOME -->
+    <section class="home about_home">
+      <h2>"Hello! Sharing world!"</h2>
+      <h1 class="about_home_title">Science Festival Spring 2022</h1>
+      <p class="about_home_text">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Lorem ipsum dolor sit amet consectetur adipisicing elit. Tenetur itaque mollitia
+        adipisci, voluptate perspiciatis pariatur aliquam laborum.
+        <span class="see_more_text">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore accusantium officiis in nisi reprehenderit vitae illum neque, eius quae
+          porro nulla. Reprehenderit libero nam quaerat autem, quas iure asperiores ad.</span
+        >
+        <br />
+      </p>
+      <p class="home_email">Please contact us per Email for any further questions about Science Festival 2022 !</p>
+      <p class="home_email email">festival2022@science.org</p>
+    </section>
+    <!-- HOME END -->
+
+    <!-- COMMONS -->
+    <section class="commons">
+      <h2>Science Festival Spring 2022 Logo</h2>
+      <hr />
+      <p>The Logo of Science Festival 2022 was decided throught the logo competition from 8. June to 7. July</p>
+      <p class="see_more_text">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatum consequuntur ducimus illum ullam laudantium consequatur quis expedita
+        veritatis architecto aliquid mollitia commodi nisi enim reprehenderit voluptates non facere, quaerat iste?
+      </p>
+      <img src="assets/images/logo_blanc.png" class="commons_logo" alt="Science Festival Logo" />
+    </section>
+    <!-- COMMONS END -->
+
+    <!-- EVENTS -->
+    <section class="events">
+      <h2>See the past Science Festival events</h2>
```

Comment:
- [ ] Make sure you follow [the design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) in this section, such as regarding **_Layout: composition and space between elements_**. Let's compare your design and the original below 👇 
- Yours here: 👇 
![seepast-1_LI](https://user-images.githubusercontent.com/67087664/146965074-057799e5-2426-49e7-b6e6-f6b9a10b4458.jpg)
- Original here: 👇 
![seepast+1_LI](https://user-images.githubusercontent.com/67087664/146965123-e4755c29-6aa4-4a46-9c1f-1e0ba310090b.jpg)
Kindly fix it 🙏

Reply:
it's done
<img width="1232" alt="Capture d’écran 2021-12-21 à 19 34 49" src="https://user-images.githubusercontent.com/7243132/146981418-81ba43f6-808e-4a02-916a-03792f9b9065.png">